src/Network/Network/NetworkSecurityPerimeter.Autorest/README.md

```diff
@@ -0,0 +1,187 @@
+  # 1. Remove the unexpanded parameter set
+  # 2. For New-* cmdlets, ViaIdentity is not required, so CreateViaIdentityExpanded is removed as well
+#  - where:
+#      variant: ^Create$|^CreateViaIdentityExpanded$
+#    remove: true
+# Remove the set-* cmdlet
+  - where:
+      verb: Set
+    remove: true
+  - where:
+      variant: ^Update$|^UpdateViaIdentity$|^UpdateViaIdentityExpanded$
+    remove: true
+  # Hide CreateViaIdentity for customization
+  - where:
+      variant: ^CreateViaIdentity$
+    hide: true
+  - where:
+      subject: NetworkSecurityPerimeter
+    set:
+      subject-prefix: ''
+  - where:
+      subject: NspProfile
+    set:
+      subject: Profile
+  - where:
+      subject: NspAccessRule
+    set:
+      subject: AccessRule
+  - where:
+      subject: NspAssociation
+    set:
+      subject: Association
+
+# Parameter Update
+# NSP
+  - where:
+      subject: NetworkSecurityPerimeter
+      parameter-name: Name
+    set:
+      parameter-name: Name
+      alias: 
+        - SecurityPerimeterName
+        - NSPName
+
+# Profile
+  - where:
+      subject: Profile
+      parameter-name: ProfileName
+    set:
+      parameter-name: Name
+      alias: 
+        - ProfileName
+
+  - where:
+      subject: Profile
+      parameter-name: NetworkSecurityPerimeterName
+    set:
+      parameter-name: SecurityPerimeterName
+      alias: 
+        - NetworkSecurityPerimeterName
+        - NSPName
+
+# AccessRule
+  - where:
+      subject: AccessRule
+      parameter-name: NetworkSecurityPerimeterName
+    set:
+      parameter-name: SecurityPerimeterName
+      alias: 
+        - NetworkSecurityPerimeterName
+        - NSPName
+
+  - where:
+      subject: AccessRule
+      parameter-name: ProfileName
+    set:
+      parameter-name: ProfileName
+      alias: 
+        - SecurityPerimeterProfileName
+        - NSPProfileName
+
+  - where:
+      subject: AccessRule
+      parameter-name: AccessRuleName
+    set:
+      parameter-name: Name
+      alias:
+        - AccessRuleName
+
+  - where:
+      subject: AccessRule
+      parameter-name: NetworkSecurityPerimeter
+    set:
+      parameter-name: NetworkSecurityPerimeters
+
+# Association
+  - where:
+      subject: Association
+      parameter-name: NetworkSecurityPerimeterName
+    set:
+      parameter-name: SecurityPerimeterName
+      alias: 
+        - NetworkSecurityPerimeterName
+        - NSPName
+
+  - where:
+      subject: Association
+      parameter-name: ProfileName
+    set:
+      parameter-name: ProfileName
+      alias: 
+        - SecurityPerimeterProfileName
+        - NSPProfileName
+
+  - where:
+      subject: Association
+      parameter-name: AssociationName
+    set:
+      parameter-name: Name
+      alias:
+        - AssociationName
+
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('if (result.NextLink != null)', 'if (result.NextLink != null && result.NextLink != "")')
```

Comment:
can you add a link to the feature request you created here as a comment?

Reply:
Added link of feature request.